pkg/plugin/volume_snapshotter_plugin.go

```diff
@@ -68,10 +82,15 @@ func (p *NewVolumeSnapshotter) Init(config map[string]string) error {
 // and with the specified type and IOPS (if using provisioned IOPS).
 func (p *NewVolumeSnapshotter) CreateVolumeFromSnapshot(snapshotID, volumeType, volumeAZ string, iops *int64) (string, error) {
 	p.Infof("CreateVolumeFromSnapshot called with snapshotID %s, volumeType %s", snapshotID, volumeType)
+	if p.clusterFlavor != constants.VSphere {
+		errMsg := "Restore of 1.0.2 vsphere volume snapshots are NOT supported in Supervisor and Guest Clusters."
```

Comment:
Do we prevent version < 1.0.2 from being installed somewhere?

Reply:
This is to handle situations where users are trying to restore vanilla <=1.0.2 backup on supervisor/guest, basically, that would be our migration scenarios.

Reply:
It reads like only 1.0.2 snapshots are not supported to be restored into supervisor and guest cluster, while actually it should <= 1.0.2 snapshots. Would you please somehow rephrase it?